components/automate-ui/src/app/services/http/http-client-auth.interceptor.spec.ts

```diff
@@ -123,6 +123,56 @@ describe('HttpClientAuthInterceptor', () => {
         });
       });
     });
+
+    describe('unfiltered flag', () => {
+      beforeEach(() => {
+        const projectList = [
+          { value: 'proj1', label: 'proj 1', checked: true },
+          { value: 'proj2', label: 'proj 2', checked: false },
+          { value: 'proj3', label: 'proj 3', checked: true },
+          { value: 'proj4', label: 'proj 4', checked: false }
+        ];
+        configure(projectList);
+        httpClient = TestBed.get(HttpClient);
+        httpMock = TestBed.get(HttpTestingController);
+        chefSession = TestBed.get(ChefSessionService);
+        spyOnProperty(chefSession, 'id_token', 'get').and.returnValue('token');
+      });
+
+      using([
+        ['does not include project header', true],
```

Comment:
fwiw, I found this abstraction made it a little trickier to read the tests and might prefer the redundancy. No biggie though.